network-api/networkapi/urls.py

```diff
@@ -48,6 +48,11 @@
         RedirectView.as_view(url='/docs/how-do-i-wagtail/'),
         name='how-do-i-wagtail'
     ),
+    re_path(
+        r'^images/([^/]*)/(\d*)/([^/]*)/[^/]*$',
+        ServeView.as_view(action='redirect'),
+        name='wagtailimages_serve',
+    ),
```

Comment:
Small nit, but let's make this its own `image_url_tag_urls.py` file with some comments that explain that this lets us use an `image_url` tag in our templates, and what it does, in the `wagtailcustomizations` dir, and then include that patthern it exports here? Just to make it much more obvious to future devs (including ourselves) that this is a customization